titiler/application/titiler/application/main.py

```diff
@@ -84,3 +84,7 @@ def landing(request: Request):
     return templates.TemplateResponse(
         name="index.html", context={"request": request}, media_type="text/html",
     )
+
+# import uvicorn
```

Comment:
Should this be here or include some conditional or comment about how to run the app locally?

Reply:
added